tlf/handle.go

```diff
@@ -414,3 +414,27 @@ func (h *Handle) DeepEqual(other Handle) bool {
 
 	return true
 }
+
+func checkUIDEquality(a, b []keybase1.UserOrTeamID) bool {
+	// Assume `a` doesn't contain any duplicates (or that we don't
+	// care if `b` matches the exact number of duplicates).
+	aMap := make(map[keybase1.UserOrTeamID]bool)
+	for _, u := range a {
+		aMap[u] = true
+	}
+	for _, u := range b {
+		if !aMap[u] {
+			return false
+		}
+		delete(aMap, u)
```

Comment:
It looks like we are assuming `b` doesn't contain any duplicate (contradictory to the comment above)?